.eslintrc

```diff
@@ -8,6 +8,7 @@
         "argsIgnorePattern": "^_"
       }
     ],
-    "react/function-component-definition": "warn"
+    "react/function-component-definition": "warn",
+    "eslint prefer-arrow-callback": ["error"]
```

Comment:
I tried various versions of this so far, but it doesnt seem to do a thing

Reply:
The `eslint` prefix is used only in comments, not in config. See [docs](https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments). Also, set it to a warning pls.

Reply:
As I wrote, I tried various ways and it does nothing. I have to have these warning disabled somehow in IDE probably or so. Have to check it still.